### ISSUE TITLE
@XmlRootElement is note expressed on enum

### ISSUE THREAD

**Issue by jskovjyskebankdk** [Jyskebank/enunciate-openapi#12]
````
@XmlRootElement(name="xmlRootElementRoleType")
@XmlType(name="xmlTypeRoleType")
@XmlEnum
public enum RootElementEnum {
	ADMIN,
	MONITOR
}
````
Is rendered as:

````
    "xml_ns0_xmlTypeRoleType":
      title: "xmlTypeRoleType"
      type: string
      enum:
      - ADMIN
      - MONITOR
````

Missing:
````
      xml:
        name: xmlRootElementRoleType
````

**Comment by jskovjyskebankdk** [comment]
Fix in AccessorDataType, awaits next Enunciate release 2.12.

**Comment by jskovjyskebankdk** [comment]
Fixed with update to enunciate 2.12.0
